### PR TITLE
List Bitzec (BZC)

### PR DESCRIPTION
**Pull request by sennevb**
- Official project URL:http://bitzec.org

- insight explorer : https://bitzec.insight.equihub.pro

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/Bitzec.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+package bisq.asset.coins;
+
+import bisq.asset.AddressValidationResult;
+import bisq.asset.AddressValidator;
+import bisq.asset.Coin;
+
+public class Bitzec extends Coin {
+
+    public Bitzec() {
+        super("Bitzec", "BZC", new BitzecAddressValidator());
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Bitzec.java]: Use RegexAddressValidator

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

**Comment by bitzec** [comment]
How to fix this

**Comment by sennevb** [comment]
@blabno code is identical to zcash, so why the regex, it's same as zcash

**Comment by blabno** [comment]
ZCash had been implemented before Regex validator was introduced.

**Comment by sennevb** [comment]
Changed to regexvalidator..

**Comment by blabno** [comment]
@sennevb Code looks good, please squash your commits into a single one (use fixup).

**Comment by sennevb** [comment]
@blabno can i chat with you somewhere, having diificulties with rebase, used google, but am stuck..

**Comment by blabno** [comment]
@sennevb https://bisq.slack.com, ping @bernard

**Comment by blabno** [comment]
ACK

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**